packages/server/lib/modes/record.js

```diff
@@ -228,7 +227,7 @@ const updateInstance = (options = {}) => {
       error,
       video,
       hooks,
-      stdout,
+      stdout: null,
```

Comment:
```suggestion
      stdout: null, // don't send stdout with the instance payload to prevent requests that are too large. stdout will later get uploaded separately anyway.
```